index.bs

```diff
@@ -4895,6 +5072,21 @@ writable stream:
     .catch(e => console.error("Something went wrong!", e));
 </code></pre>
 
+<div class="note" id="note-web-socket-wrapping-examples">
+  This specific style of wrapping a web socket interprets web socket messages directly as <a>chunks</a>.
+  This can be convenient in some cases, for example when <a>piping</a> to a <a>writable stream</a> or <a>transform
+  stream</a> for which each web socket message makes sense as a chunk to consume or transform.
+
+  However, often when people talk about "adding streams support to web sockets", they are hoping instead for the ability
```

Comment:
How about "for a new capability" instead of "for the ability"? To emphasise the distinction between wrapping an existing API, as we are doing here, and increasing the capabilities of the platform, as we'd like to do.